os/arch/arm/src/amebalite/amebalite_serial.c

```diff
@@ -818,7 +818,12 @@ static int rtl8720e_up_setup(struct uart_dev_s *dev)
 	serial_init((serial_t *) sdrv[uart_index_get(priv->tx)], priv->tx, priv->rx);
 	serial_baud(sdrv[uart_index_get(priv->tx)], priv->baud);
 	serial_format(sdrv[uart_index_get(priv->tx)], priv->bits, priv->parity, priv->stopbit);
-	serial_set_flow_control(sdrv[uart_index_get(priv->tx)], priv->FlowControl, priv->rts, priv->cts);
+	if (CONFIG_SERIAL_IFLOWCONTROL || CONFIG_SERIAL_OFLOWCONTROL) {
+		if ((sdrv[uart_index_get(priv->tx)] == 1) || (sdrv[uart_index_get(priv->tx)] == 3)) {
+			serial_set_flow_control(sdrv[uart_index_get(priv->tx)], priv->FlowControl, priv->rts, priv->cts);
+		} else {
+			printf("\nNOTE: Only UART1 & UART3 are able to use flowcontrol!\n");
```

Comment:
Please add which port tries to set flowcontrol. This can't distinguish which port has a problem with this.